cloud_images/centos7/install_prereqs.sh

```diff
@@ -26,6 +26,8 @@ echo ">>> Set up filesystem mounts"
 cat << 'EOF' > /etc/systemd/system/dcos_vol_setup.service
 [Unit]
 Description=Initial setup of volume mounts
+DefaultDependencies=no
```

Comment:
A little description on why this setting is necessary would be helpful here. Per [systemd docs](https://www.freedesktop.org/software/systemd/man/systemd.unit.html) under DefaultDependencies they say:
> Generally, only services involved with early boot or late shutdown should set this option to false. It is highly recommended to leave this option enabled for the majority of common units. If set to false, this option does not disable all implicit dependencies, just non-essential ones.

Reply:
dcos_vol_setup is formatting partitions that will be used by services that are to be started by systemd. It is better for this script to run before other systemd services are started rather than after.

Setting `DefaultDependencies=no` removes an implicit dependency `After=sysinit.target` (https://www.freedesktop.org/software/systemd/man/systemd.target.html):

> Service units, unless they set DefaultDependencies=no, automatically get a dependency on sysinit.target.

https://www.freedesktop.org/software/systemd/man/bootup.html shows where sysinit.target is activated in relation to other predefined targets and services.